src/index.js

```diff
@@ -0,0 +1,18 @@
+import './style.css';
+import { addNewTask } from './modules/addRemove.js';
+import { renderTasks } from './modules/new.js';
+// import { editTaskDescription, saveEditedTaskDescription } from './modules/edit.js';
```

Comment:
[OPTIONAL] Kindly remove  commented code to keep your code base clean

Reply:
Thanks